src/arrays.js

```diff
@@ -1,77 +1,94 @@
 const getNthElement = (index, array) => {
-  // your code here
+    if (index < array.length) {
+        return array[index]
+    } else {
+        return array[index - array.length]
+    }
+    // your code here
 };
 
 const arrayToCSVString = (array) => {
-  // your code here
+    return array.toString(array)
+        // your code here
 };
 
 const csvStringToArray = (string) => {
-  // your code here
+    return string.split(',') // your code here
 };
 
 const addToArray = (element, array) => {
-  // your code here
+    array.push(element) // your code here
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+    return array.concat(element) // your code here
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+    array.splice(index, 1)
 };
 
 const numbersToStrings = (numbers) => {
-  // your code here
+    return numbers.toString(numbers).split(',')
+        // your code here
 };
 
 const uppercaseWordsInArray = (strings) => {
-  // your code here
+    return strings.map(a => a.toUpperCase())
 };
 
 const reverseWordsInArray = (strings) => {
-  // your code here
+    return strings.map(a => a.split("").reverse().join(""))
 };
 
 const onlyEven = (numbers) => {
-  // your code here
+    return numbers.filter(num => num = 0 || num % 2 === 0)
+        // your code here
 };
 
 const removeNthElement2 = (index, array) => {
-  // your code here
+    const array2 = [...array]
+    array2.splice(index, 1)
+    return array2
```

Comment:
If you have code within a block above a `return` statement, then add a blank like above the return statement so it's clear what the function is returning:

```js
const array2 = [...array]
array2.splice(index, 1)

return array2
```